src/plugins/irc-events/message.js

```diff
@@ -87,14 +87,25 @@ module.exports = function(irc, network) {
 			highlight = network.highlightRegex.test(data.message);
 		}
 
+		const users = [];
+		const splitMessage = data.message.split(" ");
+		splitMessage.forEach((val, index) => {
+			const cleanedNick = val.replace(":", "").replace(",", "").replace("@", "").replace("'s", ""); // common nick prefixes/suffixes when chatting
+			const user = chan.findUser(cleanedNick);
```

Comment:
This doesn't sound like a great idea to do it the channel has thousands of users and its highly active.

Reply:
Does the server have a concept of rendered messages on the client? I.e. when you click “show more”/on init? Could do it there